### PR TITLE
Univariate polynomial regression

### PR DESCRIPTION
**Pull request by PrimozGodec**
It is just Univariate Polynomial Regression widget that is moved from Regression to Educational add-on.

### CODE REVIEW

**Comment by PrimozGodec** [comment]
@astaric Now it is rebased.

**Comment by astaric** [comment]
Could you also add some tests for the widget?

File: orangecontrib/educational/widgets/tests/test_owunivariateregression.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
+from Orange.widgets.tests.base import GuiTest
+from orangecontrib.educational.widgets.owunivariateregression import OWUnivariateRegression
+from Orange.data.table import Table
+from Orange.regression import LinearRegressionLearner
+
+class TestOWKmeans(GuiTest):
```
Comment (astaric) [inline orangecontrib/educational/widgets/tests/test_owunivariateregression.py]: TestOWUnivariateRegression?

**Comment by PrimozGodec** [comment]
@BlazZupan I fixed bug with learner not on output when no data. I think it should work now. May you check.

**Comment by BlazZupan** [comment]
Works fine, but I suggest to remove Apply/Apply Automatically button since it is not obvious what purpose does it serve in this widget, especially if used only for visualization.

**Comment by PrimozGodec** [comment]
I think it is ready to merge.

**Pull request merged**